infinispan-remote/cross-site-replication/minikube/LON.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  name: example-cluster-lon
+spec:
+  replicas: 2
+  expose:
+    type: NodePort
+    nodePort: 30000
+  security:
+    endpointSecretName: connect-secret-cluster-lon
+  service:
+    type: DataGrid
+    sites:
+      local:
+        name: LON
+        expose:
+          type: ClusterIP
+      locations:
+        - name: LON
```

Comment:
is this working? locations should only contain the remote sites list

Reply:
works with the operator that is currently available, but it uses 13 instead of 14. @ryanemerson can you confirm? thanks

Reply:
works well